torchvision/utils.py

```diff
@@ -380,7 +386,10 @@ def _generate_color_palette(num_masks: int):
     return [tuple((i * palette) % 255) for i in range(num_masks)]
 
 
-def _log_api_usage_once(module: str, name: str) -> None:
-    if torch.jit.is_scripting() or torch.jit.is_tracing():
+def _log_api_usage_once(obj: Any) -> None:
+    if not obj.__module__.startswith("torchvision"):
         return
-    torch._C._log_api_usage_once(f"torchvision.{module}.{name}")
+    name = obj.__class__.__name__
+    if name == "function":
+        name = obj.__name__
```

Comment:
Nit: it might be a bit cleaner to explicitly check for functions with either `inspect.isfunction()` or `isinstance(obj, types.FunctionType)`.

Reply:
+1 on the `isinstance` approach which avoids importing `inspect`.